extract_bioheart_genos_demux.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
+
+"""
+This script will
+
+- extract common variants
+- export as VCF
+
+this will be used to demultiplex scRNA-seq
+data so that we can map cells to donors.
+"""
+
+# python modules
+from cpg_utils.hail_batch import (
+    dataset_path,
+    get_config,
+    init_batch,
+    output_path,
+)
+
+import hail as hl
+from hail.methods import export_vcf
+
+__author__ = 'annacuomo'
+
+BIOHEART_JOINT_CALL_VDS = dataset_path(
+    'vds/0c401a0c6aee07549db7d85f88698fbaae4183_990.vds'
+)
+HAIL_IMAGE = get_config()['images']['scanpy']
+
+# read hail matrix table object (WGS data)
+init_batch()
+
+vds_path = BIOHEART_JOINT_CALL_VDS
+cv_demux_vcf_path = output_path('demux_vcf_common_variants.vcf.bgz')
+
+vds = hl.vds.read_vds(vds_path)
+
+# split multiallelic loci
+vds = hl.vds.split_multi(vds, filter_changed_loci=True)
+
+# densify to matrix table object
+mt = hl.vds.to_dense_mt(vds)
+
+# filter out loci & variant QC
+mt = mt.filter_rows(
+    mt.was_split  # biallelic (exclude multiallelic)
```

Comment:
Multiallelic loci should have been split already above (with `hl.vds.split_multi`), so this seems redundant

Reply:
That's what I thought as well! But when I tested in my notebook it was the was_split line that removed most of the loci, so I assume the line below was not dealing well with these? See screenshot:
<img width="712" alt="Screen Shot 2023-12-29 at 4 43 19 pm" src="https://github.com/populationgenomics/saige-tenk10k/assets/25035866/2d0db291-545f-415c-86ee-49cc5381530e">

am i missing something?

Reply:
I think it's just saying that that locus was indeed split, even though now it's biallelic. So before the split it was multiallelic, but a split happened to it when `hl.vds.split_multi` was run

Reply:
yes so now it is keeping all of these formerly multiallelic loci right, because it sees them as biallelic? Still I need to is_split filter to be the reverse, do you know what the syntax is for that?